docs/active-directory/core-concepts/01-active-directory-details-and-definitions.md

```diff
@@ -0,0 +1,111 @@
+as a tree of containers**. By virtue of a hierarchical naming system, the objects in the tree appear to be nested inside
+(contained by) other objects.”*
+
+The main components of the Active Directory logical structure are described as follows
+
+- Active Directory Forest
+    - The highest level in the hierarchy. A forest represents a collection of Directories. It is a security boundary, 
+    which gives administrators complete control and access to all the information inside both the Forest and all
+    Domains within the Forest. Multiple Forests may exist at a time.
+- Domain
+    - Domains partition the information that is stored inside the directory into smaller portions so that the
+  information can be more easily stored on various domain controllers and so that administrators have a greater
+  degree of control over replication. Data that is stored in the Active Directory is replicated throughout the forest from
+  one domain controller to another. Some data that is relevant to the entire forest is replicated to all domain
+  controllers. Other data that is relevant only to a specific domain is replicated only to domain controllers in
+  that particular domain. A good domain design makes it possible to implement an efficient replication topology.
+  This is important because it enables administrators to manage the flow of data across the network, that is, to
+  control how much data is replicated and where that replication traffic takes place.
+- Organizational Unit (OU)
+    - OU’s provide a way to group resources so that they can be managed as a single unit. This allows admins to apply
+group policies across many computers or control access of resources for many users. OUs can also be delegated to 
+specific admins, so that they only have control over their subset of resources.
+
+Other core components of an Active Directory are
+
+- DNS Support
+    - In Active Directory, DNS is the means by which directory clients locate, or discover, domain controllers. 
+      The primary components of the architecture for DNS support of Active Directory include the domain controller
+      Locator, Active Directory domain names in DNS, and Active Directory DNS objects.
+    - Every Active Directory domain has a DNS domain name (for example, [cohovineyard.com](http://cohovineyard.com/)),
+      and every domain joined computer has a DNS name (for example, 
+      [server1.cohovineyard.com](http://server1.cohovineyard.com/)). Architecturally, domains and computers are
+      represented both as objects in Active Directory and as nodes in DNS.
+- Active Directory Schema
+    - The schema contains definitions of objects which can store information in the Active Directory. It provides the 
+      attributes associated with particular resources.
+    - Different Windows versions will have different schema versions, when different versions of windows are used to 
+      comprise the Active Directory server they will agree upon a shared Schema.
```

Comment:
What format is this schema stored in? XAML? or something else?

Also think this should be discussed in another document. The fact that AD supports data validation for user inputs seems like a feature, not a core concept.

Reply:
The active directory schema is effectively the schema of the database, I haven't found documentation which point to a particular file format or a way to put the schema into a human readable text format. Users shouldn't be changing the schema anyway. Agree that it can be moved to section focusing more specifically on domains